src/Microsoft.DotNet.Build.Tasks.Packaging/src/PackageFiles/Packaging.targets

```diff
@@ -88,13 +86,6 @@
     <PlatformPackageVersion Condition="'$(PlatformPackageVersion)' == ''">1.0.1</PlatformPackageVersion>
   </PropertyGroup>
 
-  <!-- If this package explicitly sets the StableVersion then add it to the stable list -->
```

Comment:
It's not a good idea to do this since any package which depends on it will get the wrong version in its dependencies.  We aren't using StableVersion anywhere in CoreFx or WCF.  I do see some use in TFS.  I can restore this behavior to support the TFS projects but we should avoid its use.